packages/node/test/e2e/snapshots/cli.spec.js.md

```diff
@@ -6,7 +6,7 @@ Generated by [AVA](https://avajs.dev).
 
 ## "--help" prints help
 
-> output of command "/opt/homebrew/Cellar/nvm/0.40.1/versions/node/v22.11.0/bin/node /Users/boneskull/projects/lavamoat/lavamoat/packages/node/src/cli.js --help" does not match expected snapshot
```

Comment:
nice catch (thought we might have identical setup, but seems nvm likes to manage my _nodes_ in Users instead of the Cellar 🍻)

```console
➜  pwd
/opt/homebrew/Cellar/nvm/0.40.1
➜  ls
INSTALL_RECEIPT.json README.md            libexec              nvm.sh
LICENSE.md           etc                  nvm-exec             sbom.spdx.json
➜  where node
/Users/leo/.nvm/versions/node/v22.13.0/bin/node
```

Reply:
interesting, wonder why that is. I'll try not to worry about it 😄